README.md

```diff
@@ -1 +1,42 @@
-# Email-template-2
\ No newline at end of file
+# Email-template-2
```

Comment:
Good job in writing this file along with adding links to download the code.

Reply:
thanks